src/modules/tasks.js

```diff
@@ -0,0 +1,211 @@
+// eslint-disable-next-line
+import { myTasks } from '../index';
+import {
+  setDate, Flag, Status, modalStatus,
+} from './helpers';
+
+function Task(title, projectsN, comment, priority, deadline) {
+  this.title = title;
+  this.projectsN = projectsN;
+  this.comment = comment;
+  this.priority = priority;
+  this.deadline = deadline;
+  this.finished = false;
+}
+
+const first = document.getElementById('first');
+const taskForm = document.getElementById('taskForm');
+const editTaskForm = document.getElementById('editTaskForm');
+const tableArticle = document.getElementById('tableArticle');
+const todayContainer = document.getElementById('todayContainer');
+
+let countProject = false;
+// eslint-disable-next-line
```

Comment:
Kindly avoid the ESLInt inline disabling rule, it is the same as altering the ESLint config files